channel/player.go

```diff
@@ -1571,13 +1628,42 @@ func packetInventoryRemoveItem(item item) mpacket.Packet {
 	return p
 }
 
-func packetInventoryChangeEquip(char player) mpacket.Packet {
+func packetInventoryChangeEquip(chr player) mpacket.Packet {
 	p := mpacket.CreateWithOpcode(opcode.SendChannelPlayerChangeAvatar)
-	p.WriteInt32(char.id)
-	p.WriteByte(1)
-	p.WriteBytes(char.displayBytes())
```

Comment:
Why remove the use displayBytes() is there some difference between the standard display bytes and what is needed here?